tests/test_version.py

```diff
@@ -9,6 +9,11 @@ def test_get_ucx_version():
     assert ucp.core._ctx is None
 
 
+def test_git_commit_constant():
+    # __git_commit__ will only be non-empty in a built distribution
+    assert isinstance(ucp.__git_commit__, str)
+
+
 def test_version_constant():
```

Comment:
We have a test in the other RAPIDS libraries that the version string has nonzero length. Can you take a look at one of the other recent RBB PRs and do the same here?

Reply:
+1 on this suggestion.

Reply:
Yes you're right, I missed that.

Decided to just overwrite the existing tests on `__git_commit__` and `__version__` with the one we've been copying around all the other repos for the `rapids-build-backend` work.

```python
def test_version_constants_are_populated():
    # __git_commit__ will only be non-empty in a built distribution
    assert isinstance(ucp.__git_commit__, str)

    # __version__ should always be non-empty
    assert isinstance(ucp.__version__, str)
    assert len(ucp.__version__) > 0
```